Sources/Publish/API/PublishingContext.swift

```diff
@@ -138,9 +138,12 @@ public extension PublishingContext {
     /// After that, all output files and folders will be copied into the new folder.
     /// - Parameter prefix: What prefix to apply to the folder, typically
     ///   the name of the current deployment method.
+    /// - parameter targetFolderPath: Any specific subfolder path to copy the output to.
+    ///   If `nil`, then the output will be copied to the deployment folder itself.
     /// - Parameter configure: A closure used to configure the folder.
     func createDeploymentFolder(
         withPrefix prefix: String,
+        targetFolderPath: Path? = nil,
```

Comment:
I think this should be named something like `outputFolderPath` instead, to clearly show that this path is used to determine where the output should be copied.